### ISSUE TITLE
Accessibility Audit: template creation (including folder creation, moving them into different folders)

### ISSUE THREAD

**Issue by sheevdave** [GSA/notifications-admin#1940]
### Checks
- Linters etc not picking up any issues
- Page passing a WAVE and Axe Devtools scan
- Everything flowing nice when navigating with a keyboard
- Opening up Voiceover while navigating with a keyboard to make sure everything is being read out correctly and with context

**Comment by heyitsmebev** [comment]
After using WAVE and Axe Devtools to scan, there were no major errors that was flagged. 

However, when using voiceover several major issues arises 

1. When tabbing through different elements within the template creation flow, I get stuck in a loop where I cannot check  radio buttons (start with blank page or copy existing page).
2. When tabbing through different element, I cannot check any of the existing templates boxes
3. Continue button also does not work with voiceover 
4. Cancel button does work within the template creation option
![Image](https://github.com/user-attachments/assets/4d0841aa-ca5f-4ac5-9051-2e14fc42b681)

New folder button and the steps to create a new folder seems to work well. However, since I cannot check one of the current templates, I cannot move the template to the folder. Cancel and Add new folder works. 
![Image](https://github.com/user-attachments/assets/79890441-1991-4f05-a744-385df5992294)

**Comment by heyitsmebev** [comment]
merged PR
